Fixes #17290 - Usability improvements

ares opened the pull request:
I started with some UI changes some time ago, since we now have dedicated person for it and I fail to find time for doing more I decided to send current version. Feel free to take what you find useful.
![tasks_details](https://cloud.githubusercontent.com/assets/109773/18478579/12740130-79d2-11e6-8bc4-3aab75bcd843.png)

notable changes:
- owner is a link to user, reworded to triggered by
- result is only displayed if the task state is stopped, "?" otherwise, result has icon
- dates are printed more user friendly, grouped in right column
- id and label are moved to Raw tab
- progress bar changes according to patternfly
- sub tasks, parent task links converted to buttons (next to cancel)

adamruzicka commented on app/views/foreman_tasks/tasks/_details.html.erb:
```diff
@@ -79,72 +85,65 @@
+      <span class="param-name list-group-item-heading"><%= _("Result") %>:</span>
+      <span class="param-value">
+        <% if @task.state != 'stopped' %>
+          <%= content_tag(:i, '&nbsp'.html_safe, :class => 'task-status pficon-help') %>
```
Is there a way how a user could know a task got paused? If I'm reading this right having a paused task would just show a pretty question mark in the result and make the resume button clickable.

adamruzicka commented on app/views/foreman_tasks/tasks/_details.html.erb:
```diff
@@ -79,72 +85,65 @@
+    <div>
+      <span class="param-name list-group-item-heading"><%= _("Start before") %>:</span>
+      <span class="param-value"><%= @task.start_before.nil? ? '-' : time_in_words_span(@task.start_before) %></span>
+    </div>
```
How about adding another field showing how long the execution took?

adamruzicka commented:
The task name will need to be truncated

![long](https://cloud.githubusercontent.com/assets/7326770/19515734/f485b1b2-95f8-11e6-88f4-06a13b1f53c7.png)

ares commented:
Thanks @adamruzicka, name is being truncated, I moved duration to Raw tab and highlighted headers there too. I added empty line between progress bar and attributes. Please take another look.

ares commented:
I updated the label title to display the full name instead of raw label (that is to be found under raw tab). I also updated the locks page, see attached screenshot
![locks](https://cloud.githubusercontent.com/assets/109773/19934634/9eaa4760-a117-11e6-8fe5-6ef9d373f09b.png)

adamruzicka commented:
ACK from me, let's wait for Mr. Jenkins' approval and get it in

ares commented:
@adamruzicka I fixed the show vs 2pane styling. It's extracted to new css file which is included on both pages.

adamruzicka commented:
Great, let's get this in

iNecas commented:
Seems the issue with tests failures was temporary, the re-run went green http://ci.theforeman.org/job/test_plugin_pull_request/3060/

ares commented:
:champagne:
